crypto/fips1402/pom.xml

```diff
@@ -30,6 +30,24 @@
     <name>Keycloak Crypto FIPS 140-2 Integration</name>
     <description/>
 
+    <properties>
+        <maven.compiler.release>11</maven.compiler.release>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+    </properties>      
+
+<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>org.junit</groupId>
```

Comment:
So, I wanted to switch to Java 11 for this so that I could clean up the HTTP request code using the newer APIs.  But, for some reason, when I set the target and source to 11, the JUnit classes were no longer recognized in the build. To fix that, it seemed I had to switch to Junit 5, and junit-vintage is needed in order to support the existing code base. Maybe there was another way to solve this, but I wasn't able to figure it out.